test/externalTests/solc-js/solc-js.sh

```diff
@@ -35,7 +35,7 @@ function solcjs_test
     SOLCJS_INPUT_DIR="$TEST_DIR"/test/externalTests/solc-js
 
     # set up solc-js on the branch specified
-    setup master_060
+    setup smt_callback
```

Comment:
@axic Should we first merge the `solc-js` PR and then change this back to `master` actually?

Reply:
Yes, that's the plan.